app/models/service_template.rb

```diff
@@ -463,12 +463,14 @@ def update_from_options(params)
     update_attributes!(params.except(:config_info))
   end
 
-  def provision_workflow(user, options = nil)
-    options ||= {}
-    ra_options = { :target => self, :initiator => options[:initiator] }
+  def provision_workflow(user, dialog_options = nil, request_options = nil)
```

Comment:
currently `provision_workflow` is private. If you want to promote it public, you need to move up the code.